README.md

```diff
@@ -233,9 +233,13 @@ Generate time execution report for your test files. Run below command on one of
 
     # Step for Minitest
     $ KNAPSACK_GENERATE_REPORT=true bundle exec rake test
+
     # If you use rails 5 then run this instead:
     $ KNAPSACK_GENERATE_REPORT=true bundle exec rake knapsack:minitest
 
+    # If you use rails 5.1's SystemTest, run both regular and system tests
+    $ KNAPSACK_GENERATE_REPORT=true bundle exec rake test test:system
```

Comment:
double test argument `test test:system` ;)

Reply:
Nah, I think this is right! They split up `test` and `test:system`, and `test` doesn't run systemtest commands -- see the note at the bottom of this section here: http://guides.rubyonrails.org/testing.html#implementing-a-system-test

For example, this is in our rails project at DCAFEngineering:

```
$ rails -T | grep test
...
rails test                                 # Runs all tests in test folder except system ones
rails test:system                          # Run system tests only
```

So if we run the report generator command with `rake test`, it doesn't run the systemtests and then they all get kicked to node 1 (which I learned the hard way, haha).